frontend/src/components/organization/OrganizationPreviews.js

```diff
@@ -11,62 +12,57 @@ const useStyles = makeStyles({
     listStyleType: "none",
     width: "100%"
   },
-  loader: {
-    clear: "both"
+  spinner: {
+    marginTop: '48px'
   }
 });
 
 export default function OrganizationPreviews({
   organizations,
   loadFunc,
   hasMore,
-  showMembers,
```

Comment:
removing this property seems to make sense because it's not being used by any component at the moment. However if you remove it here, it would make sense to also remove it in `frontend/src/components/organization/OrganizationPreview.js` and `frontend/src/components/organization/OrganizationMetaData.js`.

Reply:
I think I missed the OrgnaziationPreview. I have pushed the update.
OrganizationMetaData didn't have this prop.